azure/app/parameters/development.template.json

```diff
@@ -94,6 +94,30 @@
         "secretName": "DfeSignInRedirectBaseUrl"
       }
     },
+    "DFE_SIGN_IN_API_CLIENT_ID": {
+      "reference": {
+        "keyVault": {
+          "id": "/subscriptions/${subscriptionId}/resourceGroups/${keyVaultResourceGroupName}/providers/Microsoft.KeyVault/vaults/${keyVaultName}"
+        },
+        "secretName": "DfeSignInApiClientId"
+      }
+    },
+    "DFE_SIGN_IN_API_SECRET": {
+      "reference": {
+        "keyVault": {
+          "id": "/subscriptions/${subscriptionId}/resourceGroups/${keyVaultResourceGroupName}/providers/Microsoft.KeyVault/vaults/${keyVaultName}"
+        },
+        "secretName": "DfeSignInApiSecret"
+      }
+    },
+    "DFE_SIGN_IN_API_ENDPOINT": {
```

Comment:
I think we need to add these to the `appSettings` in `azure/app/template.json` as well. I'm happy to add them as a commit if you like

Reply:
yep, please do!